test/e2e/resources/components/roAppInfo.brs

```diff
@@ -0,0 +1,11 @@
+sub main()
```

Comment:
Can we write a test case where we use an actual `manifest` file?

Reply:
Will do